### DISCUSSION TITLE
Updraft Section 4: Lesson 5 Tooling - case sensetive command in Written Lesson

### DISCUSSION THREAD

**Discussion by rube-de** [Cyfrin/security-and-auditing-full-course-s23#65] [Ideas]
The command in the written lesson to install is `cargo install Aderyn` this results in an error `error: could not find 'Aderyn' in registry 'crates-io' with version '*'`. (at least on mac)
Cargo seems to be case sensitive, so the correct command is `cargo install aderyn`

**Comment by cromewar** [comment]
Hello @rube-de 
thanks for this clarification, can you help us with the video on updraft or the timestamp on youtube so we can add this as a notation?

**Reply by rube-de** [reply]
It is only in the written lesson, the video shows it correct
